sophia/src/term/literal/_convert.rs

```diff
@@ -0,0 +1,495 @@
+//! Public exported by parent-module `literal`.
+//!
+
+use super::*;
+use crate::ns::xsd;
+use crate::term::{Iri, Result, StaticTerm, TermData, TermError};
+use crate::triple::stream::{SinkError, SourceError, StreamResult};
+
+/// Provide an IRI to specify the semantic datatype.
+pub trait DataType {
+    /// Static datatype of this Rust-type.
+    fn dt() -> &'static Iri<&'static str>;
```

Comment:
might as well be a `const`...
And I would rather call it `IRI`.

Reply:
The method or the trait?

Reply:
Sorry, that was ambiguous. I was suggesting to change the method `dt()` to a const `IRI`.